packages/wonder-blocks-dropdown/components/dropdown.test.js

```diff
@@ -0,0 +1,241 @@
+
+        // Act
+        dropdownCore.simulate("keydown", {keyCode: keyCodes.tab});
+        dropdownCore.simulate("keyup", {keyCode: keyCodes.tab});
+
+        // Assert
+        expect(dropdown.state("opened")).toBe(false);
+    });
+
+    it("closes itself on an external mouse click", () => {
+        // Arrange
+        const anchor = dropdown.find(IconButton);
+        anchor.simulate("click");
+
+        // Act
+        document.dispatchEvent(new MouseEvent("mouseup"));
+
+        // Assert
+        expect(dropdown.state("opened")).toBe(false);
+    });
+
+    it("open on down key when focused", () => {
+        // Arrange
+        const anchor = dropdown.find(IconButton);
+        const clickableBehavior = dropdown.find(ClickableBehavior);
+        clickableBehavior.simulate("click");
+
+        // Act
+        anchor.simulate("keydown", {keyCode: keyCodes.down});
+        anchor.simulate("keyup", {keyCode: keyCodes.down});
+
+        // Assert
+        expect(dropdown.state("opened")).toBe(true);
+    });
+
+    it.only("updates selectedValues when OptionItem clicked", () => {
+        // Arrange
+        const controlledComponent = mount(<ControlledComponent />);
+        controlledComponent.simulate("click");
+        dropdown = controlledComponent.find(DropdownCore);
+
+        // Act
+        dropdown.simulate("keydown", {keyCode: keyCodes.down});
+        dropdown.simulate("keyup", {keyCode: keyCodes.down});
+        expect(dropdown.instance().focusedIndex).toBe(0);
+
+        const optionItem = controlledComponent.find(OptionItem).at(1);
+        optionItem.simulate("click");
+
+        // Assert
+        expect(controlledComponent.state("selectedValues").length).toBe(2);
+    });
+
+    it("removes selected values from selectedValues", () => {
+        // Arrange
+        const controlledComponent = mount(<ControlledComponent />);
+        controlledComponent.simulate("click");
+        dropdown = controlledComponent.find(DropdownCore);
+
+        // Act
+        dropdown.simulate("keydown", {keyCode: keyCodes.down});
+        dropdown.simulate("keyup", {keyCode: keyCodes.down});
+        expect(dropdown.instance().focusedIndex).toBe(0);
+
+        const optionItem = controlledComponent.find(OptionItem).at(0);
+        optionItem.simulate("click");
+
+        // Assert
+        expect(controlledComponent.state("selectedValues").length).toBe(0);
+    });
+
+    it("does nothing when optionItem selected if selectedValues is null", () => {
+        // Arrange
+        const controlledComponent = mount(<ControlledComponent />);
+        controlledComponent.simulate("click");
+        controlledComponent.setState({selectedValues: null});
+        dropdown = controlledComponent.find(DropdownCore);
+
+        // Act
+        dropdown.simulate("keydown", {keyCode: keyCodes.down});
+        dropdown.simulate("keyup", {keyCode: keyCodes.down});
+        expect(dropdown.instance().focusedIndex).toBe(0);
```

Comment:
Same: remove the `expect` and move the `simulate` calls into the `Arrange` block.